ecms_base/modules/custom/ecms_workflow/src/EcmsWorkflowBundleCreate.php

```diff
@@ -51,6 +56,40 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager) {
     $this->entityTypeManager = $entityTypeManager;
   }
 
+  /**
+   * Add scheduler default settings to new content type.
+   *
+   * @param string $contentType
+   *   The machine name of the new content type.
+   *
+   * @throws \Drupal\Core\Entity\EntityStorageException
+   */
+  private function addSchedulerSettings(string $contentType): void {
+    $type = \Drupal::entityTypeManager()
+      ->getStorage('node_type')
+      ->load($contentType);
```

Comment:
You should use the injected entity manager here `$this->entityTypeManager` instead of calling the static service.